service/resources/pmd/pmd-rules.xml

```diff
@@ -13,12 +13,14 @@
         <exclude name="DoubleBraceInitialization"/>
         <exclude name="GuardLogStatement"/>
         <exclude name="JUnit4TestShouldUseBeforeAnnotation"/>
+        <exclude name="JUnit5TestShouldBePackagePrivate"/>
         <exclude name="JUnitAssertionsShouldIncludeMessage"/>
         <exclude name="JUnitTestContainsTooManyAsserts"/>
         <exclude name="JUnitTestsShouldIncludeAssert"/>
         <exclude name="PreserveStackTrace"/>
         <exclude name="SwitchStmtsShouldHaveDefault"/>
         <exclude name="UseVarargs"/>
+        <exclude name="LiteralsFirstInComparisons"/>
```

Comment:
This rule makes no sense. It was picking up code like this:

```
assertTrue(thrown.getMessage().contentEquals("Some value"));
```

There is no opportunity to do a literal first comparison in this case.
Excluding.